specs/eventing/overview.md

```diff
@@ -1,97 +1,133 @@
-  well-known input address to multiple outputs described by _Subscriptions_.
-
-<!-- This image is sourced from https://drive.google.com/open?id=10mmXzDb8S_4_ZG_hcBr7s4HPISyBqcqeJLTXLwkilRc -->
-
-![Resource Types Overview](images/resource-types-overview.svg)
-
-Channels as well as Sources are defined by independent CRDs that can be
-installed into a cluster. Both Sources and Channels implementations can be
-created directly. A Channel however offers also a way to create the backing
-implementation as part of the generic Channel (using a _channelTemplate_).
-
-## Trigger
-
-**Trigger** describes a registration of interest on a filter set of events
-delivered to a _Broker_ which should be delivered to an _Addressable_. Events
-selected by a _Trigger_ are buffered independently from other _Triggers_, even
-if they deliver to the same _Addressable_.
-
-For more details, see [Kind: Trigger](spec.md#kind-trigger).
-
-## Broker
-
-**Broker** provides an eventing mesh. This allows producers to deliver events to
-a single endpoint and not need to worry about the routing details for individual
-consumers.
-
-For more details, see [Kind: Broker](spec.md#kind-broker).
-
-## Subscription
-
-**Subscriptions** describe a flow of events from one _Channel_ to the next
-Channel\* through transformations (such as a Knative Service which processes
-CloudEvents over HTTP). A _Subscription_ controller resolves the addresses of
-transformations (`subscriber`) and destination storage (`reply`) through the
-_Callable_ and _Addressable_ interface contracts, and writes the resolved
-addresses to the _Channel_ in the `channel` reference. _Subscriptions_ do not
-need to specify both a transformation and a storage destination, but at least
-one must be provided.
-
-All event delivery linkage from a **Subscription** is 1:1 – only a single
-`channel`, `subscriber`, and `reply` may be provided.
-
-For more details, see [Kind: Subscription](spec.md#kind-subscription).
-
-## Channel
-
-**Channel** provides an event delivery mechanism which can fan out received
-events to multiple destinations via _Subscriptions_. A _Channel_ has a single
-inbound _Addressable_ interface which may accept events delivered directly or
-forwarded from multiple _Subscriptions_. Different _Channels_ may implement
-different degrees of persistence. Event delivery order is dependent on the
-backing implementation of the _Channel_.
-
-Event selection on a _Channel_ is 1:N – a single _Channel_ may fan out to
-multiple _Subscriptions_.
-
-See [Kind: Channel](spec.md#kind-channel).
-
----
-
-_Navigation_:
-
-- [Motivation and goals](motivation.md)
-- **Resource type overview**
-- [Interface contracts](interfaces.md)
-- [Object model specification](spec.md)
+The Knative Eventing API provides primitives for two common event-processing
+patterns:
+
+- Topology-based event routing ([`messaging.knative.dev`](#messaging))
+
+- Content-based event routing ([`eventing.knative.dev`](#eventing))
```

Comment:
Knowing the history here I understand why you wrote this, but w/o more context (or explaining these concepts and how they related to one another) this is hard to follow or understand why you're mentioning this. These two terms are never used again.  I'd suggest we either remove this section and just talk about the individual resources below (including how they related to each other), or go into a lot more detail about the meta-model so people understand it better.

Reply:
If you look at #25 (which includes this PR), you'll see that these are referenced there: https://github.com/knative/specs/blob/99bb2e0cb0e33f36dfd3a4475ebf08668fbca2f3/specs/eventing/control-plane.md#content-based-routing. I broke the PR up so people could get started and didn't need to digest all 1500 lines at once.